x/xos/env.go

```diff
@@ -22,10 +22,21 @@ func EnvMapToSlice(values map[string]string) []string {
 	for k, v := range values {
 		env = append(env, k+"="+v)
 	}
-	sort.Strings(env)
+	sort.Stable(sort.StringSlice(env))
 	return env
 }
 
+// EnvMapToString transform a map of key value to a string in the form "key=value;key1=value1".
+// Env vars are sorted by names to get an accurate order while testing.
+func EnvMapToString(values map[string]string) string {
```

Comment:
we can just use _EnvMapToSlice()_ inside this func, can be done in an another PR too